### PR TITLE
Using ov.save_model(...) in PTQ examples

### PR DESCRIPTION
**Pull request by alexsu52**
### Changes

Using ov.save_model(...) in PTQ examples

### Reason for changes

ov.save_model(...) is introduced in OpenVINO==2023.1.0

### Related tickets

N/A

### Tests

examples

### CODE REVIEW

**Comment by MaximProshin** [comment]
As we agreed offline, let's switch to fp16 in examples.

**Comment by MaximProshin** [comment]
`import openvino.runtime as ov`
can now be replaced by 
`import openvino as ov`

File: examples/post_training_quantization/openvino/anomaly_stfpm_quantize_with_accuracy_control/main.py [hunk 1]
```diff
@@ -165,12 +165,12 @@ def transform_fn(data_item):
 # Benchmark performance, calculate compression rate and validate accuracy
 
 fp32_ir_path = f"{ROOT}/stfpm_fp32.xml"
-ov.serialize(ov_model, fp32_ir_path)
+ov.save_model(ov_model, fp32_ir_path, compress_to_fp16=False)
```
Comment (MaximProshin) [inline examples/post_training_quantization/openvino/anomaly_stfpm_quantize_with_accuracy_control/main.py:168]: add a comment why this is required here

**Comment by MaximProshin** [comment]
@AlexKoff88 , please provide your feedback. As by default convert() doesn't return fp16 we agreed to keep as it's in the PR and add a comment why it's required to use 'compress_to_fp16=False' in the examples.

**Comment by alexsu52** [comment]
> `import openvino.runtime as ov` can now be replaced by `import openvino as ov`

done

File: examples/post_training_quantization/openvino/anomaly_stfpm_quantize_with_accuracy_control/main.py (hunk 1 above)
Comment (alexsu52) [reply examples/post_training_quantization/openvino/anomaly_stfpm_quantize_with_accuracy_control/main.py:168]: done

**Review by MaximProshin — APPROVED** [review]

**Pull request merged**